setup.cfg

```diff
@@ -1,2 +1,6 @@
 [bdist_wheel]
 universal=1
+
+[flake8]
+exclude=wsgi.py,env/*,*/migrations/*,venv/*,.env/*,.venv/*,local_settings.py,doc/*,*/node_modules/*
```

Comment:
Is this necessary? Shouldn't `pre-commit` only run on files that are checked into git?

Reply:
It shouldn't be, I just add it to all my repos. Will remove.